### PR TITLE
Data loader for CheXpert 

### PR DESCRIPTION
**Pull request by alecokas**
- Able to load the small CheXpert images along with the tabular data.
- It assumes that the data set is already downloaded since you need to ask for permission to get access to it.

### CODE REVIEW

File: utils/data_helpers.py [hunk 1]
```diff
@@ -270,7 +316,7 @@ def build_encoding_map(column):
         encoding_map[unique_value] = idx
     return encoding_map
 
-def encode_tabular_data(tab_xray_df):
+def encode_tabular_data(tab_xray_df, image_path_prefix:str):
```
Comment (Devin-Taylor) [inline utils/data_helpers.py]: why the random type definition? We should be consistent with all fxns then

File: utils/data_helpers.py [hunk 2]
```diff
@@ -282,22 +328,25 @@ def encode_tabular_data(tab_xray_df):
     for column in tab_xray_df:
         if column not in ignores and column not in normalises:
             # One-hot encode the categorical data
-            one_hot_subset_df = pd.get_dummies(tab_xray_df[column])
-            one_hot_subset_df = one_hot_subset_df.add_prefix(f'{column}_')
+            one_hot_subset_df = pd.get_dummies(tab_xray_df[column], prefix=column)
+            # one_hot_subset_df = one_hot_subset_df.add_prefix(f'{column}_')
```
Comment (Devin-Taylor) [inline utils/data_helpers.py]: just checking, meant to be commented out?

File: utils/data_helpers.py [hunk 3]
```diff
@@ -282,22 +328,25 @@ def encode_tabular_data(tab_xray_df):
...
     return encoded_df
 
-def concat_columns_into_vector(encoded_tabular_df):
-    """ Concatenate the values for all features to form an array.
-        We then pair each array / vector with the corresponding image in
-        a dictionary for easy look up.
+def remove_prefix(name, prefix: str):
+    return name[len(prefix):]
+
+def listify(encoded_tabular_df, prefix):
```
Comment (Devin-Taylor) [inline utils/data_helpers.py]: possibly change the function name? Sounds super generic but is quite specific

File: utils/data_helpers.py (hunk 1 above)
Comment (alecokas) [reply utils/data_helpers.py]: Haha yeah, I got confused when switching between projects :D

File: utils/data_helpers.py (hunk 2 above)
Comment (alecokas) [reply utils/data_helpers.py]: Yeah - I'll delete the comment. Nice spot

File: utils/data_helpers.py (hunk 3 above)
Comment (alecokas) [reply utils/data_helpers.py]: Any ideas? I struggled to come up with a decent name

Comment (Devin-Taylor) [reply utils/data_helpers.py]: extract_xray_training_data() or something along those lines? I mean, that name sucks but at least you know what is going on haha

Can just add in the comment that the returns are lists

Comment (alecokas) [reply utils/data_helpers.py]: OK, I've come up with something similar :+1:

**Pull request merged**